src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
+              "Ready": <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
+              "Image build in progress": <InProgressIcon color="var(--pf-global--palette--blue-400)" />,
+              "Building error": <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+            } 
+            return [
+              {
+                id: item.id,
+                currentStatus: item?.request.status,
+                rowArray: [
+                  item.request.name,
+                  item.request.version,
+                  item.request.distribution,
+                  item.request.type,
```

Comment:
You need to use the mapper there https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/ImageManagerDetail/constants.js#L29-L32 and replace this with `item.ImageType`.

Reply:
On a second thought use  https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/ImageManagerDetail/constants.js